src/main/scala/interfaces/Verification.scala

```diff
@@ -141,16 +142,18 @@ case class SiliconMappedCounterexample(
 ) extends SiliconCounterexample {
 
   val converter: Converter =
-    Converter(nativeModel, internalStore, heap, oldHeaps)
+    Converter(nativeModel, internalStore, heap, oldHeaps, Verifier.program)
 
   val model: Model = nativeModel
+  val interpreter: ModelInterpreter[ExtractedModelEntry, Seq[ExtractedModelEntry]] = GenericDomainInterpreter(converter)
 
   override lazy val toString: String = {
     val buf = converter.modelAtLabel
-      .map(x => s"model at label: ${x._1}\n${x._2.toString}\n")
+      .map(x => s"model at label: ${x._1}\n${ExtractedModel(x._2.entries.map(y => (y._1, interpret(y._2)))).toString}\n")
```

Comment:
I find `... map { case (foo, bar) => (foo, interpret(bar))}` to be more readable than `y._1 ... y._2` due to the names given to the two tuple components.